package.json

```diff
@@ -30,15 +30,15 @@
     "check-spelling": "cspell --config=.cspell.json \"**/*.{md,mdx,ts,mts,cts,js,cjs,mjs,tsx,jsx}\"",
     "clean": "lerna clean && lerna run clean",
     "format": "prettier --write .",
-    "generate-contributors": "yarn ts-node --transpile-only ./tools/generate-contributors.ts",
-    "generate-sponsors": "yarn ts-node --transpile-only ./tools/generate-sponsors.ts",
-    "generate-website-dts": "yarn ts-node --transpile-only ./tools/generate-website-dts.ts",
+    "generate-contributors": "node --loader ts-node/esm ./tools/generate-contributors.ts",
+    "generate-sponsors": "node --loader ts-node/esm ./tools/generate-sponsors.ts",
+    "generate-website-dts": "node --loader ts-node/esm ./tools/generate-website-dts.ts",
     "generate-lib": "nx generate-lib @typescript-eslint/scope-manager",
     "lint-fix": "eslint . --fix",
     "lint-markdown-fix": "yarn lint-markdown --fix",
     "lint-markdown": "markdownlint \"**/*.md\" --config=.markdownlint.json --ignore-path=.markdownlintignore",
     "lint": "nx run-many --target=lint --all --parallel",
-    "postinstall": "yarn ts-node --transpile-only ./tools/postinstall.ts",
+    "postinstall": "node --loader ts-node/esm ./tools/postinstall.ts",
```

Comment:
🤔 both with `yarn ts-node --transpile-only ...` and `node --loader ts-node/esm ...`, [CI fails](https://github.com/typescript-eslint/typescript-eslint/actions/runs/3653072424/jobs/6172135352) with:

```
Error: Cannot find module '../dist-raw/node-cjs-loader-utils'
Require stack:
- /home/runner/work/typescript-eslint/typescript-eslint/node_modules/ts-node/dist/index.js
- /home/runner/work/typescript-eslint/typescript-eslint/node_modules/ts-node/dist/esm.js
- /home/runner/work/typescript-eslint/typescript-eslint/node_modules/ts-node/esm.mjs
```

It doesn't repro locally for me.